### PR TITLE
[npm] suggestions for package versions

### PR DESCRIPTION
**Pull request by fwesss**
When installing a package (or anything that uses the search generator), you can add a '@' after the package name and get suggestions for package versions.

e.g. npm i fig@ ...

### CODE REVIEW

File: dev/npm.ts [hunk 1]
```diff
@@ -1,19 +1,70 @@
...
-      return JSON.parse(out).results.map(
-        (item) =>
-          ({
-            name: item.package.name,
-            description: item.package.description,
-          } as Fig.Suggestion)
-      ) as Fig.Suggestion[];
-    } catch (e) {
+      if (getVersion) {
+        return Object.keys(JSON.parse(await out(searchTerm)).versions).map(
```
[the remark this answers is not in the document]
Comment (QuiiBz) [reply dev/npm.ts:58]: Agree with @fedeci. Could it be possible to add a badge with a color, corresponding to the release type of the version? Normal releases would have a green badge, beta releases a red one, etc.

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Great idea! However, I'm unable to get the right suggestions for packages from organizations (trying to get suggestions for the `@types` organization, which is often used):

Screenshot 2021-08-28 at 11 38 20

**Comment by fedeci** [comment]
Same for `@babel/*`. I have already investigated and it is not an autocomplete fault though, but is a bug from `npms.io`. I'm going to open a PR later probably.

**Comment by fedeci** [comment]
With `@babel` namespace I can get right suggestions after I type the third letter.

**Comment by fedeci** [comment]
Reading the [API docs](url) it seems that we have to specify the `scope:` in the query.

**Pull request closed without merging**